shared/requiresSignin.ts

```diff
@@ -10,15 +12,25 @@ const publicPaths = [
 	'/api/csp-audit-report-endpoint/',
 	'/cancel-reminders/',
 	'/create-reminder/',
-	'/help-centre/',
+	{
+		path: '/help-centre/',
+		exclude: '/help-centre/diagnostic-information/',
```

Comment:
just wondering did you consider moving diagnostic information out of the help centre to avoid this?